docs/node-mixin/rules/rules.libsonnet

```diff
@@ -25,17 +25,14 @@
             ||| % $._config,
           },
           {
-            // This is CPU saturation: 1min avg run queue length / number of CPUs.
-            // Can go over 1.
-            // TODO: There are situation where a run queue >1/core is just normal and fine.
-            //       We need to clarify how to read this metric and if its usage is helpful at all.
-            record: 'instance:node_load1_per_cpu:ratio',
```

Comment:
You removed the `record` line entirely. But we need it. Obviously, the old name doesn't make sense anymore. How about `instance:node_cpu_saturation:rate%(rateInterval)s`?